C4/UI/AudioPlayer.swift

```diff
@@ -48,7 +48,7 @@ public class AudioPlayer: NSObject, AVAudioPlayerDelegate {
     /// ````
     public init?(_ name: String) {
         do {
-            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback)
+			try AVAudioSession.sharedInstance().setCategory(.playback)
```

Comment:
All these indentation changes need to be reverted.

Reply:
Oops. Moved 'em back to spaces.